vignettes/Surveys_Walkthrough.Rmd

```diff
@@ -129,12 +127,13 @@ There are different ways to sample populations for surveys. The most commonly us
 1. **Simple random sampling (SRS)**: requires a comprehensive sampling frame (i.e. total list of households inside a refugee camp or 
 GPS based sampling in a known area).
 2. **Cluster based sampling**: most commonly used in combination with sampling villages proportional to population size.
-3. **Stratified samples**: uses either SRS or cluster based sampling but in two different adminstrative areas in order to obtain
+3. **Stratified samples**: uses either SRS or cluster based sampling but in two different administrative areas in order to obtain
 precise prevalence estimates for them.
 
-The templates can accomodate analysis of data from all sampling designs.
+The templates can accommodate analysis of data from all of these sampling designs.
 
 ### Sample size
+
```

Comment:
worth a note that there is a handbook chapter coming on this?

Reply:
done